pages/[id]/edit.js

```diff
@@ -20,6 +29,7 @@ export default function EditPage({ defaultTags }) {
       });
 
       if (response.ok) {
+        setUpdated(true);
```

Comment:
I think we can call the notify method here directly. 

```suggestion
        notify();
```